webpack/JobWizard/JobWizard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Wizard } from '@patternfly/react-core';
```

Comment:
The wizard shows a "cancel" button. What will it do?

Reply:
I think it should navigate to the previous page, so if the user was in the jobs table it would go there, and if they were in the hosts page it would go there

Reply:
Sounds good. Is it doable?

Reply:
Currently I'm using `history.goBack()` so it should work